DQM/SiStripMonitorApproximateCluster/plugins/BuildFile.xml

```diff
@@ -0,0 +1,7 @@
+<use name="FWCore/Framework"/>
+<use name="FWCore/PluginManager"/>
+<use name="FWCore/ParameterSet"/>
+<use name="DQMServices/Core"/>
```

Comment:
If kept as separate package, the dependencies to `DataFormats/Common` and `DataFormats/SiStripCluster` are missing.

Reply:
Addressed at https://github.com/cms-sw/cmssw/pull/40295/commits/a90f8e55543f72001dfd4eafd3a64d2a4b362cd5